.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+name: build
+
+on: [push, pull_request]
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```

Comment:
How does this combine with pull_request above? (line 3)

Reply:
You would have to make a PR _from_ master which will never occur. The PR trigger is for when we add regular builds to this file and turn down Circle builds.